core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/common/upgrade-common.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Remove jq installed from directly from github
+  file:
+    path: /usr/local/bin/jq
+    state: absent
+  register: jq
+
+- name: Install jq Debian family packages
+  apt:
+    name:
+      - jq
+    update_cache: yes
+    state: present
+  when: ansible_os_family == "Debian"
+
+- name: Install jq RedHat family packages
+  yum:
+    name:
+      - jq
+    update_cache: yes
+    state: present
+  when: ansible_os_family == "RedHat"
```

Comment:
I think you can use "package" ansible module since name of the package is the same.